src/libraries/System.Private.CoreLib/src/System/Guid.Unix.cs

```diff
@@ -33,5 +33,24 @@ public static unsafe Guid NewGuid()
 
             return g;
         }
+
+        private static unsafe Guid CreateRandomizedPartialVersion7()
+        {
+            Guid g;
```

Comment:
is it ok for UUIDv7 that you use a non-initialized variable here so the upper 6 bytes will be zero or stack garbage? (since corelib is compiled with SkipLocalsInit)